docs/decisions/ADR001_TechnologyDecisions.md

```diff
@@ -0,0 +1,39 @@
+
+status: proposed 
+date: 2022-11-01
+deciders: Kavya, Susmitha, Uma, Miguel
+
+# Technology Selection decision for HeyBlue App
+
+## Context and Problem Statement
+
+This ADR defines the Technology choices made to develop the HeyBlue Mobile and Web Application.
+The goal of this Web/Mobile application is to provide a cost effective platform which helps the civilians connect with police officers for a positive interaction experience at a scale in a secure way. 
+
+## Decision Drivers
+
+* Building a cost effective application
+* Extendable and Flexibility to integrate with other technologies
+* Hige development community
+
+## Considered Options
+
+* Opt for Open Source Technologies
+
+## Decision Outcome
+
+Chosen option: "Opt for Open Source Technologies, because we are developing a Charity Application where the costs should be kept as low as possible. Saving cost is our critical criteria.
+
+
+## Pros and Cons of the Options
+
+### Opt for Open Source Technologies
+
+* Good, because opting for Open Source software enables flexibility to integrate with other technologies
+* Good, because it is cost effective, stable and performant with large development community support.
+* Good, because No Vendor locking
+* Good, because it offers cost savings
+* Bad, because of the security treats, however adopting a secured underlying framework.
```

Comment:
Threats is misspelled.

I'm not sure I understand this point.  All software is at risk of security vulnerabilities.  Also it's not clear what is meant by "adopting a secured underlying framework"